test/parsers_test.cpp

```diff
@@ -267,6 +267,7 @@ void test_config_file(const char* config_file)
 
     const char content1[] =
     " gv1 = 0#asd\n"
+    "; semi comment\n"
```

Comment:
A also want to see a commented-out line which has an assignment, and a check to ensure the value is not assigned.